apps/financial-aid/backend/src/app/modules/application/dto/updateApplication.dto.ts

```diff
@@ -2,14 +2,22 @@ import { IsNotEmpty, IsString, IsOptional, IsNumber } from 'class-validator'
 
 import { ApiProperty } from '@nestjs/swagger'
 
-import { ApplicationState } from '@island.is/financial-aid/shared/lib'
+import {
+  ApplicationEventType,
+  ApplicationState,
+} from '@island.is/financial-aid/shared/lib'
 
 export class UpdateApplicationDto {
   @IsNotEmpty()
   @IsString()
   @ApiProperty()
   readonly state: ApplicationState
 
+  @IsNotEmpty()
+  @IsString()
+  @ApiProperty()
+  readonly event: ApplicationEventType
+
```

Comment:
Do we want application event not to be optional? 🙃

Reply:
Talked about in IRL and we are keeping as not empty for now.